polygeist/tools/cgeist/Lib/Attributes.cc

```diff
@@ -382,7 +385,7 @@ AttrBuilder::addPassThroughRawIntAttr(llvm::Attribute::AttrKind Kind,
   OpBuilder Builder(&Ctx);
   NamedAttribute NamedAttr(
       StringAttr::get(&Ctx, llvm::Attribute::getNameFromAttrKind(Kind)),
-      Builder.getIntegerAttr(Builder.getIntegerType(64), Value));
+      StringAttr::get(&Ctx, Twine(Value)));
```

Comment:
When translating to LLVMIR, `forwardPassthroughAttributes` expects arrays within 'passthrough' to contain two strings.